src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -103,11 +103,9 @@ function tryRemoveElement(elem) {
         });
     }
 
-    function hidePrePlaybackPage() {
-        const animatedPage = document.querySelector('.page:not(.hide)');
-        animatedPage.classList.add('hide');
+    function hideScroll() {
         // At this point, we must hide the scrollbar placeholder, so it's not being displayed while the item is being loaded
-        document.body.classList.remove('force-scroll');
+        document.body.classList.add('hide-scroll');
```

Comment:
It's probably better to inline this instead of having a one line function.

Reply:
It gets called twice for each file and tells you clearly what it is doing.